### PR TITLE
add securityContext for KES pods in Tenant CRD

### PR DESCRIPTION
**Pull request by pjuarezd**

### CODE REVIEW

**Review by dvaldivia — DISMISSED** [review]
LGTM

File: pkg/resources/statefulsets/kes-statefulset.go [hunk 1]
```diff
@@ -142,6 +143,49 @@ func kesSecurityContext(t *miniov2.Tenant) *corev1.PodSecurityContext {
...
+	// This is specially needed in OpenShift where Security Context Constraints restrict them
+	// if let empty then OCP can pick the values from the constraints defined.
+	containerSecurityContext := corev1.SecurityContext{}
+	runAsNonRoot := true
+	var runAsUser int64 = 1000
+	var runAsGroup int64 = 1000
+	poolSCSet := false
+
+	// Values from pool.SecurityContext ONLY if provided
+	if t.Spec.KES.SecurityContext != nil {
```
Comment (shtripat) [inline pkg/resources/statefulsets/kes-statefulset.go:159]: Should we add `t.Spec.KES != nil` as well here?

Comment (pjuarezd) [reply pkg/resources/statefulsets/kes-statefulset.go]: Done

**Review by jiuker — APPROVED** [review]

**Review by shtripat — APPROVED** [review]
lgtm

**Pull request merged**